jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt

```diff
@@ -243,15 +246,22 @@ class GettingStartedPanel(private val project: Project) : BorderLayoutPanel() {
                             label(message("codewhisperer.gettingstarted.panel.licence_comment")).applyToComponent { foreground = PanelConstants.TEXT_FONTCOLOR }
                         }
                         row {
-                            browserLink(message("aws.onboarding.getstarted.panel.login_with_iam"), url = CODEWHISPERER_LEARN_MORE_URI)
+                            text(message("aws.onboarding.getstarted.panel.login_with_iam")) { hyperlinkEvent ->
+                                val actionEvent = AnActionEvent.createFromInputEvent(
+                                    hyperlinkEvent.inputEvent,
+                                    SETUP_AUTH_DIALOG,
+                                    null
+                                ) { if (PlatformDataKeys.PROJECT.`is`(it)) project else null }
+                                ActionManager.getInstance().getAction("aws.toolkit.AuthDialogAction").actionPerformed(actionEvent)
```

Comment:
all the action infrastructure is completely unnecessary for this

```suggestion
            SetupAuthenticationDialog(e.getRequiredData(LangDataKeys.PROJECT)).show()
```

Reply:
Addressed it